src/helm/benchmark/adaptation/adapters/in_context_learning_adapter.py

```diff
@@ -218,10 +218,15 @@ def construct_example_prompt(self, instance: Instance, include_output: bool, ref
 
         # References (optionally) and output
         output: str
+
+        delimiter = ","
         if reference_index is None:
             # Put only the correct reference as the output
-            correct_reference: Optional[Reference] = instance.first_correct_reference
-            output = correct_reference.output.text if correct_reference is not None else "n/a"
+            correct_references: List[Reference] = instance.all_correct_references
+            if not correct_references:
+                output = "n/a"
+            else:
+                output = delimiter.join([correct_reference.output.text for correct_reference in correct_references])
```

Comment:
Note to self: This might change instances if we have scenarios with multiple correct references somehow (which should not happen).